### PR TITLE
Setup helm chart releases site

### PR DESCRIPTION
**Pull request by addyess**
Add Job to build helm charts on pushes to `rockcraft` branch

### CODE REVIEW

File: .github/workflows/release_charts.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - rockcraft
+
```
Comment (berkayoz) [inline .github/workflows/release_charts.yaml]:
```suggestion
    paths:
      - "deploy/charts/**"
```

This should make sure we do/trigger a release only if something is changed under `deploy/charts`.

File: .github/workflows/release_charts.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+jobs:
+  release:
+    permissions:
```
Comment (berkayoz) [inline .github/workflows/release_charts.yaml]: Not sure if this is necessary I think we got away without having this before, see [here](https://github.com/canonical/hostpath-provisioner/blob/master/.github/workflows/release-chart.yaml)

**Review by berkayoz — COMMENTED** [review]
Great work overall, some small comments.

File: .github/workflows/release_charts.yaml (hunk 2 above)
Comment (addyess) [reply .github/workflows/release_charts.yaml]: ah -- didn't know.  I was just following 
https://helm.sh/docs/howto/chart_releaser_action/#github-actions-workflow

**Comment by addyess** [comment]
@berkayoz 
is the [`cr.yaml`](https://github.com/canonical/hostpath-provisioner/blob/master/.github/cr.yaml) necessary as well?

**Comment by berkayoz** [comment]
@addyess
Not unless you want to change the default release name. Especially to not have conflicts if the project name is the same as the helm chart name, so we can have 2 types of releases. One for the project itself binaries etc, and a separate one for helm charts.

**Comment by addyess** [comment]
@berkayoz cool, i'm ready to test if you are

**Review by berkayoz — APPROVED** [review]

**Pull request merged**